src/pages/graphql/schema/checkout/queries/get-payment-config.md

```diff
@@ -0,0 +1,322 @@
+                        "tagline": false
+                    }
+                },
+                "apple_pay": {
+                    "code": "payment_services_paypal_apple_pay",
+                    "title": "Apple Pay",
+                    "payment_intent": "authorize",
+                    "sort_order": "10",
+                    "sdk_params": [
+                        {
+                            "name": "src",
+                            "value": "https:\/\/www.paypal.com\/sdk\/js?client-id=..."
+                        }
+                    ],
+                    "is_visible": true,
+                    "button_styles": {
+                        "layout": "vertical",
+                        "color": "gold",
+                        "shape": "rect",
+                        "height": null,
+                        "label": "paypal",
+                        "tagline": false
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Input attributes
+
+The `getPaymentConfig` object will differ on which attributes must contain depending on the payment method.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`location` | PaymentLocation! | The origin location for that payment request. The possible values are PRODUCT_DETAIL, MINICART, CART, CHECKOUT, ADMIN
+
+## Output attributes
+
+The `PaymentConfigOutput` contains details about each configured payment method:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`ApplePayConfig` | String! | ApplePay payment method configuration
+`HostedFieldsConfig` | String! | PayPal Hosted fields payment method configuration
+`SmartButtonsConfig` | String! | PayPal Smart Buttons payment method configuration
+
+## `PaymentConfigItem` interface
+
+The `PaymentConfigItem` interface contains the fields that are common to all the payment methods. This interface contains the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`code` | String! | The payment method code as defined in the payment gateway
+`is_visible` | Boolean | Indicates whether the payment method is shown
+`payment_intent` | String | Defines the payment intent. The possible values are `Authorize` or `Capture`
+`sdk_params` | SDKParams | PayPal parameters required to load the PayPal JavaScript SDK
+`sort_order` | String | Defines the preference for the sorting order of the payment method
+`title` | String! | The display name of the payment method
+
+This interface also has the following implementations:
+
+* ApplePayConfig
+* HostedFieldsConfig
+* SmartButtonsConfig
+
+Each implementation contains objects with specific attributes for each of tthese payment methods.
+
+### `ApplePayConfig` attributes
+
+The `ApplePayConfig` payment method configuration has a `button_styles` object containing the following attributes:
+
+Attribute | Data Type | Description
+--- | --- | ---
+`color` | String | The button color
+`height` | Units | The button height
```

Comment:
```suggestion
`height` | Int | The button height
```

By units, I meant "The button height, in pixels", or whatever the correct unit is.

Reply:
aaah, thanks

Reply:
Cannot find this information anywhere, I'll ask the team meanwhile

Reply:
great